src/error/errorCode.ts

```diff
@@ -19,6 +19,8 @@ export enum ErrorCode {
     TriggerBuildOnWorkspaceWithMultipleFolders = 'TriggerBuildOnWorkspaceWithMultipleFolders',
     RunDocfxFailed = 'RunDocfxFailed',
     GenerateReportFailed = 'GenerateReportFailed',
+    UndefinedUserBuild = 'UndefinedUserBuild',
```

Comment:
what is this error code used for?

Reply:
forgot to remove it after move the undefined user check logic to extension.ts, removed